### ISSUE TITLE
Disappearing Configuration

### ISSUE THREAD

**Issue by clee2005** [YoYoGames/GameMaker-Bugs#5332]
### Description
I have an Amazon configuration I use for our Amazon apps, and upon upgrading to 2024.2.0.132 I lost the default setting and was only seeing the Amazon config... so the bundle id and version number were gone and I almost released it with the wrong details.   

After resetting it, I've noticed that going back into the project after closing it can show the same results... that is, the default is lost and only shows the Amazon setting... then I set them both appropriately, and I can switch back and forth as expected, but it will revert after closing the project and reopeing.

### Steps To Reproduce
Add a configuration and make some changes (I'm working with Android).   Then make changes in the Amazon config.   Switch back and forth and see it work as expected.  Close the project, and reopen to see only the AMazon config under both Default and Amazon.

### Which version of GameMaker are you reporting this issue for?
IDE v2024.2.0.132 Runtime v2024.2.0.163

### Which operating system(s) are you seeing the problem on?
Windows 10.0.22631.0

[f31cde15-af88-4901-ac88-50673bbaa7f3](https://api.gamemaker.io/api/github/downloads/f31cde15-af88-4901-ac88-50673bbaa7f3)

**Comment by clee2005** [comment]
To be clear... I was changing the BundleId under the General Tab, and the Google Licencing public key under the Packaging tab.   I imagine other fields are affected as well, but those are the two that I noticed and was able to reproduce.

**Comment by clee2005** [comment]
Oh boy... It seems this bug also affects the extensions.  When switching back and forth in the configs the entire DEFAULT configuration is overwritten with one of the sub configurations.   Argh!

**Comment by stuckie** [comment]
Duplicate of https://github.com/YoYoGames/YoYoStudio/issues/9779

**Comment by stuckie** [comment]
If your configuration is set to default on closing, it should not cause a problem.. it should also be fixed in current April betas.

**Comment by clee2005** [comment]
Hi Steven,

Duplicate of YoYoGames/YoYoStudio#9779
<https://github.com/YoYoGames/YoYoStudio/issues/9779> doesn't link to
anything... 404

Closing on DEFAULT does NOT solve the issue.   I just played with it, and
noticed that switching to "Amazon" and making changes, and then coming back
to "Default" it looks good.   Then I tried to build (and this is the 2nd
time it's happened), the build had the Amazon version and bundle ID.    I
then ensured it was on "Default" and closed the IDE, reopened, and
"Default" was showing the Amazon stuff.
